### PR TITLE
🎈 11.2.1

### PR DESCRIPTION
**Pull request by chapulina**
# 🎈 Release

Preparation for 11.2.1 release.

Comparison to 11.2.0: https://github.com/ignitionrobotics/sdformat/compare/sdformat11_11.2.0...sdf11

Needed to fix ABI break: https://github.com/ignitionrobotics/ign-gazebo/issues/891

## Checklist
- [x] Asked team if this is a good time for a release
- [ ] There are no changes to be ported from the previous major version
- [ ] No PRs targeted at this major version are close to getting in
- [ ] Bumped minor for new features, patch for bug fixes
- [ ] Updated changelog
- [x] Updated migration guide (as needed)
- [ ] Link to PR updating dependency versions in appropriate repository in [ignition-release](https://github.com/ignition-release) (as needed): <LINK>

**Note to maintainers**: Remember to use **Squash-Merge**

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -2,6 +2,14 @@
 
 ### libsdformat 11.X.X (202X-XX-XX)
 
+### libsdformat 11.2.0 (2021-06-28)
```
Comment (scpeters) [inline Changelog.md:5]: 11.2.1

**Review by scpeters — CHANGES_REQUESTED** [review]
we need to bump the version in the root CMakeLists.txt as well

File: Changelog.md (hunk 1 above)
Comment (scpeters) [reply Changelog.md]: ce2c652

**Comment by scpeters** [comment]
> we need to bump the version in the root CMakeLists.txt as well

ce2c652

**Review by scpeters — APPROVED** [review]

**Review by azeey — APPROVED** [review]

**Pull request merged**